xds/internal/balancer/ringhash/ringhash.go

```diff
@@ -147,8 +156,10 @@ func (sc *subConn) effectiveState() connectivity.State {
 // it's Connect() will be triggered. If the SubConn state is already Idle, it
 // will just call Connect().
 func (sc *subConn) queueConnect() {
+	fmt.Printf(" ===== queue connect for %v\n", sc)
```

Comment:
Delete (there are at least a few of these in here)

Reply:
Done